tests/scripts/components.sh

```diff
@@ -1,53 +0,0 @@
-# components.sh
-#
-# Copyright The Mbed TLS Contributors
-# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
-
-# This file contains the test components that are executed by all.sh
-
-# The functions below are named as follows:
```

Comment:
Feels a shame to be loosing that comment (except the last paragraphs which is obsolete as pointed out previously). Perhaps add it back in `all.sh` right before sourcing the `component-xxx.sh` files?

Reply:
That was a temporary duplication. The comment is still present in `all.sh`.